go/slack/slack.go

```diff
@@ -0,0 +1,59 @@
+package slack
+
+import (
+	"fmt"
+	"github.com/slack-go/slack"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+)
+
+type conf struct {
+	Token   string `yaml:"slack_api_token"`
+	Channel string `yaml:"slack_channel"`
+}
```

Comment:
Small note: this will probably be changed in the future, Cobra integrates well with [Viper](https://github.com/spf13/viper).